### ISSUE TITLE
[Bug] "Copy System Logs" fails to locate .logarchive

### ISSUE THREAD

**Issue by Bogidon** [fastlane/fastlane#8850]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

After upgrading to Xcode 8.3, [`FastlaneCore::DeviceManager#copy_logarchive`](https://github.com/fastlane/fastlane/blob/master/fastlane_core/lib/fastlane_core/device_manager.rb#L252) stopped working. It threw errors that it wasn't able to find the exported `.logarchive`. I checked and indeed the log archive was not there. (The log collection logic was last modified in #8125)

I'm not 100% confident this is an Xcode 8.3 error specifically. Also, I'm running on CircleCI, so it could be that I lack necessary permissions (apparently you need root permissions for `log collect` on macOS, not sure how that works through `simctl spawn`). 

As a temporary fix, I'm passing `include_simulator_logs: false` to scan. My local env is currently kaput, so I'll be able to confirm whether it's Circle's fault or not only after I get things running locally again. Will update when I have additional info. 

##### Complete output when running fastlane, including the stack trace and command used
```
[15:34:08]: ------------------------------
[15:34:08]: --- Collecting system logs ---
[15:34:08]: ------------------------------
[15:34:08]: $ xcrun simctl getenv 7C307D61-9549-405A-8F90-32A7566C80CB SIMULATOR_SHARED_RESOURCES_DIRECTORY 2>/dev/null
[15:34:08]: $ xcrun simctl spawn 7C307D61-9549-405A-8F90-32A7566C80CB log collect 2>/dev/null
[15:34:11]: No such file or directory @ rb_file_s_stat - /Users/distiller/Library/Developer/CoreSimulator/Devices/7C307D61-9549-405A-8F90-32A7566C80CB/data/system_logs.logarchive/.
...
bundler: failed to load command: fastlane (/Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/bin/fastlane)
Errno::ENOENT: No such file or directory @ rb_file_s_stat - /Users/distiller/Library/Developer/CoreSimulator/Devices/7C307D61-9549-405A-8F90-32A7566C80CB/data/system_logs.logarchive/.
  /Users/distiller/.rubies/ruby-2.4.1/lib/ruby/2.4.0/fileutils.rb:1224:in `stat'
  /Users/distiller/.rubies/ruby-2.4.1/lib/ruby/2.4.0/fileutils.rb:1224:in `lstat'
  /Users/distiller/.rubies/ruby-2.4.1/lib/ruby/2.4.0/fileutils.rb:1253:in `copy'
  /Users/distiller/.rubies/ruby-2.4.1/lib/ruby/2.4.0/fileutils.rb:415:in `block in copy_entry'
  /Users/distiller/.rubies/ruby-2.4.1/lib/ruby/2.4.0/fileutils.rb:1387:in `wrap_traverse'
  /Users/distiller/.rubies/ruby-2.4.1/lib/ruby/2.4.0/fileutils.rb:412:in `copy_entry'
  /Users/distiller/.rubies/ruby-2.4.1/lib/ruby/2.4.0/fileutils.rb:390:in `block in cp_r'
  /Users/distiller/.rubies/ruby-2.4.1/lib/ruby/2.4.0/fileutils.rb:1460:in `block in fu_each_src_dest'
  /Users/distiller/.rubies/ruby-2.4.1/lib/ruby/2.4.0/fileutils.rb:1474:in `fu_each_src_dest0'
  /Users/distiller/.rubies/ruby-2.4.1/lib/ruby/2.4.0/fileutils.rb:1458:in `fu_each_src_dest'
  /Users/distiller/.rubies/ruby-2.4.1/lib/ruby/2.4.0/fileutils.rb:389:in `cp_r'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane_core/lib/fastlane_core/device_manager.rb:265:in `copy_logarchive'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane_core/lib/fastlane_core/device_manager.rb:232:in `copy_logs'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/scan/lib/scan/runner.rb:101:in `block in copy_simulator_logs'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/scan/lib/scan/runner.rb:99:in `each'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/scan/lib/scan/runner.rb:99:in `copy_simulator_logs'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/scan/lib/scan/runner.rb:82:in `handle_results'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/scan/lib/scan/runner.rb:9:in `run'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/scan/lib/scan/manager.rb:14:in `work'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/actions/scan.rb:22:in `run'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/runner.rb:252:in `block (2 levels) in execute_action'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/runner.rb:230:in `block in execute_action'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/runner.rb:226:in `chdir'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/runner.rb:226:in `execute_action'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/runner.rb:148:in `trigger_action_by_name'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/fast_file.rb:146:in `method_missing'
  lanes/test/test_lane.rb:3:in `block in parsing_binding'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/lane.rb:33:in `call'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/runner.rb:45:in `chdir'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/runner.rb:45:in `execute'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/lane_manager.rb:52:in `cruise_lane'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/command_line_handler.rb:30:in `handle'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/commands_generator.rb:104:in `block (2 levels) in run'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/commander-fastlane-4.4.4/lib/commander/command.rb:178:in `call'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/commander-fastlane-4.4.4/lib/commander/command.rb:153:in `run'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/commander-fastlane-4.4.4/lib/commander/runner.rb:476:in `run_active_command'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:38:in `run!'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/commander-fastlane-4.4.4/lib/commander/delegates.rb:15:in `run!'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/commands_generator.rb:303:in `run'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/commands_generator.rb:42:in `start'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/cli_tools_distributor.rb:65:in `take_off'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/gems/fastlane-2.25.0/bin/fastlane:20:in `<top (required)>'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/bin/fastlane:22:in `load'
  /Users/distiller/apple-teespring/vendor/bundle/ruby/2.4.0/bin/fastlane:22:in `<top (required)>'
```

### Environment

<details><summary>🚫 fastlane environment 🚫</summary>

### Stack

| Key                         | Value                                                   |
| --------------------------- | ------------------------------------------------------- |
| OS                          | 10.12.3                                                 |
| Ruby                        | 2.4.1                                                   |
| Bundler?                    | true                                                    |
| Git                         | git version 2.11.0 (Apple Git-81)                       |
| Installation Source         | ~/apple-teespring/vendor/bundle/ruby/2.4.0/bin/fastlane |
| Host                        | Mac OS X 10.12.3 (16D32)                                |
| Ruby Lib Dir                | ~/.rubies/ruby-2.4.1/lib                                |
| OpenSSL Version             | OpenSSL 1.0.2k  26 Jan 2017                             |
| Is contained                | false                                                   |
| Is homebrew                 | false                                                   |
| Is installed via Fabric.app | false                                                   |
| Xcode Path                  | /Applications/Xcode-8.3.app/Contents/Developer/         |
| Xcode Version               | 8.3                                                     |

### System Locale

| Error                       |
| --------------------------- |
| No Locale with UTF8 found 🚫 |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
fastlane_require 'aws-sdk'
fastlane_require 'pry'

before_all do |lane, options|
  #-----------------ENVS-------------#
  $additional_match_app_identifier_distribution = ENV['ADDITIONAL_MATCH_APP_IDENTIFIER_DISTRIBUTION']
  $additional_match_app_identifier_local        = ENV['ADDITIONAL_MATCH_APP_IDENTIFIER_LOCAL']
  $additional_match_app_identifier_staging      = ENV['ADDITIONAL_MATCH_APP_IDENTIFIER_STAGING']
  $app_identifier_distribution                  = ENV['APP_IDENTIFIER_DISTRIBUTION']
  $app_identifier_local                         = ENV['APP_IDENTIFIER_LOCAL']
  $app_identifier_staging                       = ENV['APP_IDENTIFIER_STAGING']
  $app_name                                     = ENV['APP_NAME']
  $apple_id                                     = ENV['APPLE_ID']
  $aws_access_key_id                            = ENV['AWS_ACCESS_KEY_ID']
  $aws_secret_key                               = ENV['AWS_SECRET_KEY']
  $aws_secrets_bucket_name                      = ENV['AWS_SECRETS_BUCKET_NAME']
  $aws_secrets_bucket_region                    = ENV['AWS_SECRETS_BUCKET_REGION']
  $aws_secrets_certificates_path                = ENV['AWS_SECRETS_CERTIFICATES_PATH']
  $aws_secrets_configurations_path              = ENV['AWS_SECRETS_CONFIGURATIONS_PATH']
  $branch_certificates_distribution             = ENV['BRANCH_CERTIFICATES_DISTRIBUTION']
  $branch_certificates_local                    = ENV['BRANCH_CERTIFICATES_LOCAL']
  $branch_certificates_staging                  = ENV['BRANCH_CERTIFICATES_STAGING']
  $certificates_repo                            = ENV['CERTIFICATES_REPO']
  $changelog_path                               = ENV['CHANGELOG_PATH']
  $configurations_output_path                   = ENV['CONFIGURATIONS_OUTPUT_PATH']
  $crashlytics_api_token                        = ENV['CRASHLYTICS_API_TOKEN']
  $crashlytics_build_secret                     = ENV['CRASHLYTICS_BUILD_SECRET']
  $no_specs_tag                                 = ENV['NO_SPECS_TAG'] || "[NOSPECS]"
  $release_git_prefix                           = ENV['RELEASE_GIT_PREFIX']
  $scheme_distribution                          = ENV['SCHEME_DISTRIBUTION']
  $scheme_local                                 = ENV['SCHEME_LOCAL']
  $scheme_staging                               = ENV['SCHEME_STAGING']
  $slack_app_specific_updates_channel           = ENV['SLACK_APP_SPECIFIC_UPDATES_CHANNEL']
  $slack_developers_updates_channel             = ENV['SLACK_DEVELOPERS_UPDATES_CHANNEL']
  $slack_release_updates_channel                = ENV['SLACK_RELEASE_UPDATES_CHANNEL']
  $target_name                                  = ENV['TARGET_NAME']
  $test_summary_output_directory                = ENV['CIRCLE_TEST_REPORTS'] || "/dev/null"
  $xcodeproj_name                               = ENV['XCODEPROJ_NAME']
  $xcodeproj_path                               = ENV['XCODEPROJ_PATH']

  #---------------Computed----------#
  $ipa_path                 = "build/#{$app_name}.ipa"
  $configurations_yml_path  = "configurations/#{$aws_secrets_configurations_path}"

  #---------------Fastlane----------#
  ENV['FASTLANE_USER']      = $apple_id
  ENV['MATCH_GIT_URL']      = $certificates_repo


  #-----------------AWS-------------#
  Aws.config.update({
    region: $aws_secrets_bucket_region,
    credentials: Aws::Credentials.new($aws_access_key_id, $aws_secret_key)
  })

  $bucket = Aws::S3::Resource.new.bucket($aws_secrets_bucket_name)
end

# Import lanes
Dir.glob("lanes/**/*_lane.rb") { |file| import file }

# Skip generating README.md
skip_docs
```
</details>

**No Appfile found**

### fastlane gems

| Gem      | Version | Update-Status      |
| -------- | ------- | ------------------ |
| fastlane | 2.25.0  | 🚫 Update available |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.1.0        |
| bundler                   | 1.14.6       |
| io-console                | 0.4.6        |
| CFPropertyList            | 2.3.5        |
| i18n                      | 0.8.1        |
| minitest                  | 5.10.1       |
| thread_safe               | 0.3.6        |
| tzinfo                    | 1.2.3        |
| activesupport             | 4.2.8        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.1        |
| aws-sigv4                 | 1.0.0        |
| jmespath                  | 1.3.1        |
| aws-sdk-core              | 2.8.11       |
| aws-sdk-resources         | 2.8.11       |
| aws-sdk                   | 2.8.11       |
| babosa                    | 1.0.2        |
| claide                    | 1.0.1        |
| coderay                   | 1.1.1        |
| colored                   | 1.2          |
| colored2                  | 3.1.2        |
| colorize                  | 0.8.1        |
| highline                  | 1.7.8        |
| commander-fastlane        | 4.4.4        |
| unf_ext                   | 0.0.7.2      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20170223 |
| dotenv                    | 2.2.0        |
| excon                     | 0.55.0       |
| multipart-post            | 2.0.0        |
| faraday                   | 0.12.0.1     |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| faraday_middleware        | 0.11.0.1     |
| fastimage                 | 2.1.0        |
| gh_inspector              | 1.0.3        |
| jwt                       | 1.5.6        |
| little-plugger            | 1.1.4        |
| multi_json                | 1.12.1       |
| logging                   | 2.2.0        |
| memoist                   | 0.15.0       |
| os                        | 0.9.6        |
| signet                    | 0.7.3        |
| googleauth                | 0.5.1        |
| httpclient                | 2.8.3        |
| hurley                    | 0.2          |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| uber                      | 0.0.15       |
| representable             | 2.3.0        |
| retriable                 | 2.1.0        |
| google-api-client         | 0.9.28       |
| json                      | 2.0.3        |
| mini_magick               | 4.5.1        |
| multi_xml                 | 0.6.0        |
| plist                     | 3.2.0        |
| rubyzip                   | 1.2.1        |
| security                  | 0.1.3        |
| slack-notifier            | 1.5.1        |
| terminal-notifier         | 1.7.1        |
| unicode-display_width     | 1.1.3        |
| terminal-table            | 1.7.3        |
| tty-screen                | 0.5.0        |
| word_wrap                 | 1.0.0        |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.4.3        |
| rouge                     | 1.11.1       |
| xcpretty                  | 0.2.6        |
| xcpretty-travis-formatter | 0.0.4        |
| method_source             | 0.8.2        |
| slop                      | 3.6.0        |
| pry                       | 0.10.4       |
</details>

*generated on:* **2017-04-11**
</details>
